src/locales/en.json

```diff
@@ -97,7 +97,8 @@
   "hints": {
     "linearElement": "Click to start multiple points, drag for single line",
     "linearElementMulti": "Click on last point or press Escape or Enter to finish",
-    "resize": "You can constraint proportions by holding SHIFT while resizing"
+    "resize": "You can constraint proportions by holding SHIFT while resizing",
+    "rotate": "You can constraint angles by holding SHIFT while rotating"
```

Comment:
I found it odd this was using the noun `constraint` rather than the verb `constrain`

Reply:
Oops, maybe we fix the "resize" message in another PR, and this PR will follow that? Would you or somebody be willing to create a PR?

Reply:
```suggestion
    "resize": "You can constrain proportions by holding SHIFT while resizing",
    "rotate": "You can constrain angles by holding SHIFT while rotating"
```